test/_globalHooks.js

```diff
@@ -1,6 +1,7 @@
 import lockfile from 'lockfile';
 
-afterEach(function globalAfterEach() {
+/* eslint-disable mocha/no-top-level-hooks */
```

Comment:
I think it makes more sense to disable for the whole file rather than a block, because this is the file where we define these exceptions.